apps/web/app/api/storage/route.ts

```diff
@@ -0,0 +1,334 @@
+import { authOptions } from "@/app/api/auth/[...nextauth]/authOptions";
+import { hasUserEnvironmentAccess } from "@/lib/api/apiHelper";
+import { getServerSession } from "next-auth";
+import { NextRequest, NextResponse } from "next/server";
+import {
+  getFileFromLocalStorage,
+  getFileFromS3,
+  putFileToLocalStorage,
+  putFileToS3,
+} from "@formbricks/lib/services/storage";
+import path from "path";
+
+const UPLOADS_DIR = path.resolve("./uploads");
+
+export async function GET(req: NextRequest) {
+  const searchParams = req.nextUrl.searchParams;
+
+  const fileName = searchParams.get("fileName");
+
+  if (!fileName) {
+    return NextResponse.json(
+      {
+        success: false,
+        message: "Missing required parameters",
```

Comment:
Please provide more information to the user; which required parameter is missing. see other api endpoints for reference.